### PR TITLE
networkmanager: add pixel test for edit checkboxes

### PR DESCRIPTION
**Pull request by jelly**
Closes: #19019

---

It now looks as following:

image

### CODE REVIEW

File: pkg/networkmanager/network-interface.jsx [hunk 1]
```diff
@@ -264,7 +264,7 @@ export const NetworkInterfacePage = ({
         return (
             <DescriptionListGroup>
                 <DescriptionListTerm>{_("Status")}</DescriptionListTerm>
-                <DescriptionListDescription>
+                <DescriptionListDescription className="networking-interface-status">
```
Comment (cockpituous) [inline pkg/networkmanager/network-interface.jsx:267]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19023-20230626-214120-185a4bfd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by jelly** [comment]
Hmm I just found this https://github.com/cockpit-project/cockpit/commit/5e8444c9745367f94c294f1a3ad1002f39b99abf#diff-f532f3dbbc8a77b7edbabbafb2778bd1c04c7a1b64b1ff5870a5f3880133fa37R113 which makes me think my solution might need to be re-adjusted?

**Review by garrett — CHANGES_REQUESTED** [review]
Yeah, this should not need a one-off fix. It looks like a PF related problem?

**Comment by jelly** [comment]
> Yeah, this should not need a one-off fix. It looks like a PF related problem?

Maybe it is because this used to be a `span` and an `input` next to each other but now it's  a `span` and a `<div><input>`

**Comment by jelly** [comment]
Final pixel test changes are mostly around buttons which seem fine https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230703-145202-8ab5302f-fedora-38/log.html

**Review by KKoukiou — APPROVED** [review]

**Comment by martinpitt** [comment]
@jelly Not sure if you noticed, but there's pixel test failures -- the IPv6 address isn't stable, so needs to be ignored.

**Comment by jelly** [comment]
> @jelly Not sure if you noticed, but there's pixel test failures -- the IPv6 address isn't stable, so needs to be ignored.

No, it seems we have a double spacing now?!!

**Review by martinpitt — APPROVED** [review]
Cheers! This was quite embarrassing indeed.

File: pkg/networkmanager/networking.scss [hunk 2]
```diff
@@ -111,7 +111,7 @@ th {
 }
 
 // If the button is not the first item in the DescriptionListDescription add left padding
-.network-interface-settings dd button:not(:first-child) {
+.network-interface-settings div.pf-c-description-list__text button:not(:first-child) {
```
Comment (garrett) [inline pkg/networkmanager/networking.scss:114]: This is wrong. Never use div or span when selecting things in CSS. Also, usually try to not prefix with the element when selecting a class. (PF has famously changed elements several times, including recently.)

**Review by garrett — DISMISSED** [review]
Comment inline

**Comment by garrett** [comment]
I wasn't aware of this and fixed it as a commit in the PF5 PR @ https://github.com/cockpit-project/cockpit/pull/19111/commits/aa5e628917ccc91002b3445785aaea823a62d3c0

**Comment by martinpitt** [comment]
FWIW, https://github.com/cockpit-project/cockpit/pull/19111/commits/aa5e628917ccc91002b3445785aaea823a62d3c0 seems fine to me as well. @jelly are you d'accord with closing this PR then?

**Comment by martinpitt** [comment]
Yes, seems fine to rebase this PR after #19111 for adding the pixel test.

**Comment by jelly** [comment]
> Yes, seems fine to rebase this PR after #19111 for adding the pixel test.

That is now, let's rebase :)

**Comment by jelly** [comment]
Added pixel tests and only triggered required tests.

**Review by martinpitt — APPROVED** [review]
Pixel perfect! Thanks for adding this.

**Comment by jelly** [comment]
> Cheers! This was quite embarrassing indeed.

Needed to re-introduce a class to exclude the random ipv6 address from getting excluded from the pixel comparison.

**Review by martinpitt — APPROVED** [review]
Dankjewel!

**Pull request merged**